modules/nf-core/mitohifi/mitohifi/main.nf

```diff
@@ -0,0 +1,63 @@
+process MITOHIFI_MITOHIFI {
+    tag "$meta.id"
+    label 'process_high'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "MitoHiFi module does not support Conda. Please use Docker / Singularity instead."
+    }
+
+    // Docker image available at the biocontainers Dockerhub
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://ghcr.io/marcelauliano/mitohifi:master':
+        'ghcr.io/marcelauliano/mitohifi:master' }"
```

Comment:
Since it is the same container, please add like https://github.com/nf-core/modules/blob/499265b27ba02a9805761c5c980556f2525d52dd/modules/nf-core/fastk/fastk/main.nf#L6